technology-considerations/API_management_AppSync_GraphQL_REST.md

```diff
@@ -0,0 +1,62 @@
+# Options for API management
+
+We want to have an intermediate component between our frontend (which consists of the browser and the Play web application)
+and the database. 
```

Comment:
Worth mentioning that the file checking background jobs will also need to call the API? It means that we have to write and maintain at least two API clients. The background jobs will also use different authentication to the frontend.